kiwi/system/root_import/docker.py

```diff
@@ -63,10 +63,19 @@ def sync_data(self):
         )
         synchronizer.sync_data(options=['-a', '-H', '-X', '-A'])
 
+        # A copy of the uncompressed image and its checksum are
+        # kept inside the root_dir in order to ensure the later steps
+        # i.e. system create are atomic and don't need any third
+        # party archive.
+        self._copy_image(self.uncompressed_image)
+
     def __del__(self):
-        if self.oci_layout_dir:
-            Path.wipe(self.oci_layout_dir)
-        if self.oci_unpack_dir:
-            Path.wipe(self.oci_unpack_dir)
-        if self.uncompressed_image:
-            Path.wipe(self.uncompressed_image)
+        try:
+            if self.oci_layout_dir:
+                Path.wipe(self.oci_layout_dir)
+            if self.oci_unpack_dir:
+                Path.wipe(self.oci_unpack_dir)
+            if self.uncompressed_image:
+                Path.wipe(self.uncompressed_image)
+        except Exception:
+            pass
```

Comment:
All right `except/pass` this needs to be changed, as it can miss behave under some circumstances.

Just as side comment, note that `try/except/pass` block has nothing to do with the `Path.wipe`, I included it because this delete code gets executed when an Exception is thrown during the __init__ of the base class, in which case the `post_init` method of the derived class doesn't get executed meaning that the attributes evaluated on that method (`self.oci_unpack_dir` and so on) are not initialized at that point.

Just explaining it because you'll see that arranging that I will also refactor a little the root_import init method of the base classe, just in case you wonder why.